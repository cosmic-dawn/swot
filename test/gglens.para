#Configuration file for swot - gg lensing example
#----------------------------------------------------------#
#Input catalogues                                          #
#----------------------------------------------------------#
data1          data/lenses.cat  # input data catalogue #1
cols1          1,2,3,4	        # Column ids for data1
data2          data/sources.cat # input data catalogue #2
cols2          1,2,3,4,5,6,7    # Column ids for data2
#----------------------------------------------------------#
#Correlation options                                       #
#----------------------------------------------------------#
corr           gglens	 # Type of correlation: [auto,cross,gglens,auto_wp,cross_wp]
range          40e-3,5	 # Correlation range. Dimension same as "proj":
nbins          10	 # Number of bins
log            yes	 # Logarithmic bins [yes,no]
err            jackknife # Resampling method [bootstrap,jackknife]
nsamples       32	 # Number of samples for resampling (power of 2)
OA             0.05	 # Open angle for approximation (value or "no") 
#----------------------------------------------------------#
#Cosmology (for gal-gal correlations, w(R) and xi(rp,PI))  #
#----------------------------------------------------------#
H0             72	 # Hubble parameter
Omega_M        0.258	 # Relative matter density
Omega_L        0.742	 # Relative energy density (Lambda)
deltaz         0.03	 # For gg lensing: Zsource > Zlens + deltaz
#----------------------------------------------------------#
#Output options                                            #
#----------------------------------------------------------#
proj           como	 # Axis projection [theta, como, phys:
                    	 # auto, cross: default: theta (deg)
                    	 # auto_wp, cross_wp, gglens: default: como (Mpc)
out            corr.out	 # Output file
cov            no	 # Covariance matrix of errors [yes,no] (in "out".cov)
