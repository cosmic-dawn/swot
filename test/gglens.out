#Gal-gal lensing. Sigma(R) vs R, linear approximation
#Resampling: jackknife (32 samples)
#Coordinates: comoving
#Cosmolgy: H0 = 72, Omega_M = 0.258, Omega_L = 0.742
#  R(Mpc)  SigR(Msun/pc^2) err(weights) err(resampling) Nsources       <R>          e2
   0.0509220   85.8552845   18.9565131   17.2674439            1130    0.0537460   -3.2475459
   0.0825271   50.9469783   10.7139650   18.0560160            3179    0.0859329    5.3556312
   0.1337481   23.8878126    6.4667989   15.4168274            8345    0.1387727   15.4178665
   0.2167597   12.7605156    4.0068420    5.0265788           21637    0.2257924    3.2772853
   0.3512930   13.1202662    2.4494888    3.8939131           57442    0.3646489    0.4263038
   0.5693253    8.9254119    1.5068360    2.1054117          150289    0.5914947    2.2406203
   0.9226808    4.9047963    0.9361988    1.3461157          388273    0.9576156   -0.4647586
   1.4953488    3.1690041    0.5821443    1.0231865         1001801    1.5524595   -0.2022461
   2.4234469    2.3468515    0.3642798    0.7602086         2549907    2.5146455    0.2790534
   3.9275752    1.0223321    0.2288992    0.7349761         6444172    4.0740269   -0.3067849
